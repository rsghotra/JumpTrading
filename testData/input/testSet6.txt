0, 100000, 0, 2, 1075
0, 100001, 0, 3, 1080
0, 100002, 0, -5, 1080
0, 100003, 1, 5, 1050
0, 100004, 1, 0, 1050
1, 100003