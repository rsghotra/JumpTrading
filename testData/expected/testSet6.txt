2,3,1080
4,100003,2
3,100001
2,2,1075
3,100003
3,100000
